packages/mainmenu/src/mainmenu.ts

```diff
@@ -311,6 +311,9 @@ export class MainMenu extends MenuBar implements IMainMenu {
     if (label) {
       menu.title.label = translator.__(label);
     }
+    if (!menu.title.label) {
+      menu.title.label = translator.__(Private.capitalize(options.id));
```

Comment:
I do not understand what this is trying to @krassowski  🤔

Reply:
This fallback to using `id` when label is not given was already there. I did not add it - just moved around and added the translate wrapper. See https://github.com/jupyterlab/jupyterlab/pull/10739/files#diff-88177d2aa7480e9e525ae52ef585dc98ea673735196459630dd65d41f6173938L61

Reply:
Ok, but having the trans there on a string that is capitalized probably will never work.

Reply:
It can work if it is used elsewhere. Of course it will not get extracted. I agree that this fallback should not need to exist, but is this wrong to wrap it into the `trans.__` if if is already there (compared to the alternative of not doing that)?

Reply:
🤷🏼